src/components/application_manager/include/application_manager/resumption/resumption_data_processor_impl.h

```diff
@@ -134,6 +134,7 @@ class ResumptionDataProcessorImpl
   bool IsResumptionFinished(const uint32_t app_id,
```

Comment:
@ychernysheva you may also rename this function to `void EraseProcessedRequest` and use only your version of `IsResumptionFinished` in both places